### PR TITLE
PLATUI-848: Uplift dependencies

### PR DESCRIPTION
**Pull request by JoPintoPaul**

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.18
+sbt.version=1.4.3
```
Comment (matthewmascord) [inline project/build.properties]: I've gone with 1.4.6 on the other ones?

File: project/plugins.sbt [hunk 2]
```diff
@@ -4,14 +4,14 @@ resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
...
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.9.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.13.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.1.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-play-cross-compilation" % "0.20.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-play-cross-compilation" % "1.2.0")
 
 val playPlugin = sys.env.getOrElse("PLAY_VERSION", "2.6") match {
```
Comment (matthewmascord) [inline project/plugins.sbt]: Should the default be 2.7 now?

File: project/build.properties (hunk 1 above)
Comment (JoPintoPaul) [reply project/build.properties]: @matthewmascord Uplifted to 1.4.6

File: project/plugins.sbt (hunk 2 above)
Comment (JoPintoPaul) [reply project/plugins.sbt]: @matthewmascord Updated to use 2.7 as the default

**Review by matthewmascord — APPROVED** [review]

File: build.sbt [hunk 3]
```diff
@@ -12,12 +14,12 @@ lazy val playLanguage = (project in file("."))
...
-    crossScalaVersions := Seq("2.11.12", "2.12.10"),
+    crossScalaVersions := scalaVersions,
     PlayCrossCompilation.playCrossCompilationSettings,
     libraryDependencies ++= AppDependencies.all,
     resolvers += Resolver.typesafeRepo("releases"),
     resolvers += Resolver.bintrayRepo("hmrc", "releases")
   )
   .settings(sourceDirectories in(Compile, TwirlKeys.compileTemplates) +=
-    (sourceDirectory in Compile).value / "play-26")
+    (sourceDirectory in Compile).value / playDir)
```
Comment (matthewmascord) [inline build.sbt:24]: @JoPintoPaul I think this is the problem

**Pull request merged**